bravado/fido_client.py

```diff
@@ -148,4 +149,8 @@ def result(self, timeout=None):
             return self._eventual_result.wait(timeout=timeout)
         except crochet.TimeoutError:
             self._eventual_result.cancel()
-            raise
+            raise fido.exceptions.HTTPTimeoutError(
```

Comment:
Alright: just pointing that this was never about not doing it, just doing it as a reraise.

Anyway python 3 ftw so let's go